aws/code/Dockerfile

```diff
@@ -1,27 +1,27 @@
-FROM rapidsai/rapidsai-cloud-ml:latest 
+FROM rapidsai/rapidsai-core:22.12-cuda11.5-runtime-ubuntu18.04-py3.9 
```

Comment:
This looks great, was there much difference between the cloud-ml container and the regular one in this example? It looks like you just needed to install `dask-ml`, was that all you needed?

Reply:
I added these lines in the dockerfile: 
```
   pip3 install sagemaker-training cupy-cuda115 flask dask-ml \
   pip3 install --upgrade protobuf
```